scripts/HoveredMidiRelative/HoveredMidiRelativeExt.py

```
@@ -516,119 +626,164 @@ def onReceiveMidiSlotLearn(self, index: int):
+			old_default = self.activePar.default
+			new_default = self.activePar.eval()
+			
+			# Apply the change
+			self.activePar.default = new_default
+			
+			# Create undo action
+			self.undo_manager.create_set_default_undo(self.activePar, old_default, new_default)
+			
+			# update display
+			self.display_manager.update_parameter_display(self.activePar, bottom_text = '_DEF_')
+		except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)